### PR TITLE
docs: Remove non-existent subdomains

### PR DESCRIPTION
**Pull request by minisbett**
# Describe your changes

Removed weird subdomains that do not even exist. AFAIK some of the c1-c6 can be removed too, not sure when what is used. I ran my server fine for a year with just c, ce and c4 but jupiter said he needed smth like c5 so idk.

## Checklist
- [x] The changes pass pre-commit checks (`make lint`)
- [x] The changes follow coding style

### CODE REVIEW

**Comment by 0i8** [comment]
c4-5-6 are still used by the client in some way, i still get requests from it

**Comment by minisbett** [comment]
> c4-5-6 are still used by the client in some way, i still get requests from it

ye I remembered that some high numbers are used, I ran on c4 and possibly c6 I don't remember but never seen c1-c3.

**Comment by cmyui** [comment]
i believe `i` and `s` are used - `c1` `c2` `c3` are unused, not sure about `c5` and `c6`.

**Comment by tsunyoku** [comment]
s and i are used by the client and are exposed by bpy (in nginx). you need them back

**Comment by tsunyoku** [comment]
c4, ce and c6 are used alongside c for bancho. other c<num> are unused

**Comment by minisbett** [comment]
> s and i are used by the client and are exposed by bpy (in nginx). you need them back

fr? what for? never seen any of them used anywhere

**Comment by minisbett** [comment]
> > > s and i are used by the client and are exposed by bpy (in nginx). you need them back
> > 
> > 
> > fr? what for? never seen any of them used anywhere
> 
> thumbnails and audio previews (direct)

ohhhh right didn't think about that

**Comment by tsunyoku** [comment]
also it's b that is used, not i, mb

b & s must stay. i is a ripple relic

**Comment by minisbett** [comment]
The readme didn't even contain b until now lol

**Comment by cmyui** [comment]
can we update the image which shows these as well?

**Comment by minisbett** [comment]
sure

**Comment by minisbett** [comment]
fixed

File: .github/images/ssl_cf_6.png
[the remark this answers is not in the document]
Comment (0i8) [reply .github/images/ssl_cf_6.png:1]: why

[the remark this answers is not in the document]
Comment (minisbett) [reply .github/images/ssl_cf_6.png:1]: why

**Comment by cmyui** [comment]
reopen without memes if you want it to be merged

**Pull request closed without merging**